Formula/d/dosbox-staging.rb

```diff
@@ -36,6 +36,7 @@ class DosboxStaging < Formula
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on "fluid-synth"
+  depends_on "gettext"
```

Comment:
`gettext` should almost never be a runtime dependency on Linux (only needed if formula uses commands like `msgfmt` at runtime as main libraries are part of glibc).

On macOS, from old logs it looks like only Ventura, which means we are probably hitting the `ld` bug. 

In case that Ventura is only impacted, I would recommend not adding dependency. Should either fix issue in `brew` or do manual workaround of `-ld_classic` so that `b_asneeded=true` (https://mesonbuild.com/Builtin-options.html#base-options) works as expected.

Reply:
yeah, it is only ventura builds recording the linkage thing, see https://github.com/Homebrew/homebrew-core/actions/runs/9863188002/job/27237049521#step:3:862

I think we can just note it and move forward